### ISSUE TITLE
Uploading of Training-Data Request needs to be enhanced

### ISSUE THREAD

**Issue by bjoekeldude** [skunkforce/omniview#43]
In order for the THGA to find out, which Auto-Shop uploaded how many Scans, there needs to be an additional field in the interface and the REST function, transmitting a Workshop-ID. This should be a UUID and be given out by the THGA to each auto-shop.
An empty auto-shop-UUID (ASID) is permitted.

![image](https://github.com/skunkforce/omniview/assets/10761141/96e018ed-9193-4874-8854-1e277bfeb060)

**Comment by AKMaily** [comment]
Is this UUID implemented in the HUB API ? @DanielNowak98

**Comment by DanielNowak98** [comment]
> Is this UUID implemented in the HUB API ? @DanielNowak98

The UUID is the unique ID for any workshop, right?

**Comment by AKMaily** [comment]
Yes it is a unique ID that the workshops should get from the THGA so they can find out which data is send from which workshop

**Comment by DanielNowak98** [comment]
> Yes it is a unique ID that the workshops should get from the THGA so they can find out which data is send from which workshop

Ok, I think this should be covered in the HUB. I'll find out.

**Comment by AKMaily** [comment]
@DanielNowak98 could you find out which endpoint is used in the HUB API for requesting the UUID?
